openadapt/app/cards.py

```diff
@@ -1,16 +1,26 @@
 import signal
```

Comment:
```
from subprocess import Popen
import signal

from nicegui import ui

from openadapt.app.objects.local_file_picker import LocalFilePicker
from openadapt.app.util import get_scrub, set_dark, set_scrub, sync_switch
```
from before import

Reply:
wait.. we do it that way? I think it looks better the other way but I'll change it

Reply:
[5fe7100](https://github.com/OpenAdaptAI/OpenAdapt/pull/375/commits/5fe71003e36d9aea4c5799417428662dcdf81d2b)